syntax = "proto3";

message LightStatus {
    float status = 1;
}

service Light {
    rpc turnOn(LightStatus) returns (LightStatus) {}
    rpc turnOff(LightStatus) returns (LightStatus) {}
}